internal/pkg/table/path.go

```diff
@@ -133,13 +133,14 @@ type Validation struct {
 }
 
 type Path struct {
-	info      *originInfo
-	parent    *Path
-	pathAttrs []bgp.PathAttributeInterface
-	dels      []bgp.BGPAttrType
-	attrsHash uint32
-	aslooped  bool
-	reason    BestPathReason
+	info         *originInfo
+	parent       *Path
+	pathAttrs    []bgp.PathAttributeInterface
+	dels         []bgp.BGPAttrType
+	attrsHash    uint32
+	aslooped     bool
+	reason       BestPathReason
+	receiveVrfId uint32 //VRF in which the path was received.
```

Comment:
Don't fatten Path structure. Can you create a map or whatever in zclient.go to store that information?

Reply:
Thank you for reply and reviewing my source code. 
I introduced receiveVrfId to Path struct referring PR #1766 without careful consideration. 
I sent new source code on this PR for introducing map on zclient instead of deleting receiveVrfID from Path struct.